### ISSUE TITLE
[Bug]: Product data not making it to the PDP

### ISSUE THREAD

**Issue by gideonco** [vuestorefront/shopify#144]
### Expected Behavior

While navigating to the PDP I expect product content to be displayed

### Actual Behavior

**Cannot read property '0' of undefined**

<img width="1787" alt="Screen Shot 2022-01-10 at 7 07 18 PM" src="https://user-images.githubusercontent.com/1274123/148858641-9241605f-a664-4a3b-9255-df3a309c1dfe.png">

Tracing this error a bit, some product data is passed from shopify to enhanceProduct but it does not pass this step.  It doesn't seem to be in the expected schema.

### Possible Solution

I have fixed this locally as an immediate patch by rolling back shopify-buy to "2.13.0" in `@vue-storefront/shopify-api`

### Steps to reproduce

- Generate a shopify project using the CLI
- Add test credentials
- Navigate to any PDP

### What version of shopify integration are you using?

v1.0.9

### What version of Node.js are you using?

v14.17.3

### What browser (and version) are you using?

Chrome 97

### What operating system (and version) are you using?

macOS

### Relevant log output

```shell
server.js:24349:385 src/helpers/internals/enhanceProduct.ts:8:enhanceProduct 
src/getters/productGetters.ts:78:Object.getProductFiltered [as getFiltered]
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jeffpdotone** [comment]
In the meantime please do the downgrade this is actually one of the tasks we have in our roadmap to improve the "enhanceProduct" method.

**Comment by jeffpdotone** [comment]
Related to vuestorefront/vue-storefront#6600 and it's being addressed with this PR #146

**Comment by jeffpdotone** [comment]
Fixed by vuestorefront/shopify#146
